ideascube/tests/test_models.py

```diff
@@ -37,13 +37,17 @@ def test_create_superuser():
 def test_list_users():
     model = get_user_model()
     model.objects.create_user('123456')
-    model.objects.create_user('__system__')
+
+    unfiltered_queryset = model.objects.get_queryset_unfiltered()
+    if not len(unfiltered_queryset.filter(serial='__system__')):
```

Comment:
I prefer:

```python
if not unfiltered_queryset.filter(serial='__system__').exists():
```

It's much more explicit what you're actually checking.

However, why not just use the `systemuser` fixture? :)